### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by anniejiwon**
## TODO

- [ ] Documentation
- [ ] Tests
- [ ] Experimental evidence (required for Masonry changes)
- [ ] Accessibility checkup

### CODE REVIEW

**Comment by anniejiwon** [comment]
screen shot 2018-11-29 at 5 48 38 pm

**Comment by anniejiwon** [comment]
screen shot 2018-11-29 at 5 48 50 pm

File: packages/gestalt/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gestalt",
-  "version": "0.84.0",
+  "version": "0.84.1",
```
Comment (pfarejowicz) [inline packages/gestalt/package.json]: no need for this, we can do a bump/release in a separate PR

File: packages/gestalt/src/icons/index.js [hunk 2]
```diff
@@ -77,7 +81,9 @@ import smileyOutline from './smiley-outline.svg';
 import sound from './sound.svg';
 import speech from './speech.svg';
 import speechEllipsis from './speech-ellipsis.svg';
+import switchAccount from './switch.svg';
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/index.js]: should we rename the file to match the name?

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 0b19098b5f955c0952356b24d4303cd45236255e

https://deploy-preview-425--gestalt.netlify.com

File: packages/gestalt/src/icons/index.js [hunk 3]
```diff
@@ -77,7 +81,9 @@ import smileyOutline from './smiley-outline.svg';
 import sound from './sound.svg';
 import speech from './speech.svg';
 import speechEllipsis from './speech-ellipsis.svg';
+import switchAccount from './switchAccount.svg';
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/index.js]: I think we follow the `switch-account.svg` naming convention for these

File: packages/gestalt/src/icons/ad.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M7 20h10v-4H7v4zM17 1H7a3 3 0 0 0-3 3v16a3 3 0 0 0 3 3h10a3 3 0 0 0 3-3V4a3 3 0 0 0-3-3z" fill="#8E8E8E" fill-rule="evenodd"/></svg>
```
Comment (pfarejowicz) [inline packages/gestalt/src/icons/ad.svg]: i dont see this one in the `index.js` file

**Pull request merged**